app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int) -> None:

```

Comment:
One attribute per line, when you have a lot of attributes. At checklist you have a good example of styling
```    
    def long_function_name(
            var_one,
            var_two,
            var_three,
            var_four
    ) -> None:
```